erna/scripts/process_fact_data.py

```diff
@@ -34,7 +34,7 @@ def make_jobs(jar, xml, db_path, output_directory, df_mapping,  engine, queue, v
 @click.argument('data_dir', type=click.Path(exists=True, dir_okay=True, file_okay=False, readable=True) )
 @click.argument('jar', type=click.Path(exists=True, dir_okay=False, file_okay=True, readable=True) )
 @click.argument('xml', type=click.Path(exists=True, dir_okay=False, file_okay=True, readable=True) )
-@click.argument('db', type=click.Path(exists=True, dir_okay=False, file_okay=True, readable=True) )
+@click.argument('db', type=click.Path(exists=True, dir_okay=True, file_okay=True, readable=True) )
```

Comment:
Why got this changed? The db should not be a directory, right?

Reply:
I think, @tarrox changed it, in order to use the fact-tools auxservice that is going through the auxfile directory tree. Anyways, I think this should be renamed.

Reply:
and documented. Probably by @mackaiver as he was the first author.

Reply:
It can be a dictionary since the new auxfileservice is able to work with the aux dictionary structure.

Reply:
A dictionary, do you mean a directory?

So then it's not a db anymore, right? So maybe call it somthing like `aux_source`?

Reply:
Currently it is called db everywhere (in the arguments, in the erna xml files and so on). I would rather open a issue of it and let it be fixed as one to make sure every instance is changed. As it would require a change in more than one file.

Reply:
But before it **was** a db. Because you changed it, it's something else. So it needs to be done in this PR.

Reply:
Well, actually it was never a db but the path to e.g. a db-file that was handed over to a fact-tools process.
If it needs to be changed, it has to be changed in fact-tools in the erna-example-xml and in erna in each "process_..." script (which is 4 files if I am not mistaken). I think that's doable. I think, both of your points (1. It has been changed here, so it has to be changed here, 2. lets make it in a separate PR, so its done more thoroughly) are generally valid. However, the afford is small, so I'd say, lets do it in this PR and now.

Reply:
Actually, I will do it now!

Reply:
should be handled by #43